### PR TITLE
References on Software Ecosystems

### PR DESCRIPTION
**Pull request by pmbittner**

### CODE REVIEW

File: literature.bib [hunk 1]
```diff
@@ -4853,6 +4887,18 @@ @article{MJ:TAOSD14
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2014 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{BPT+:IST14,
+	author = {Thorsten Berger and Rolf{-}Helge Pfeiffer and Reinhard Tartler and Steffen Dienst and Krzysztof Czarnecki and Andrzej Wasowski and Steven She},
+	title = {Variability mechanisms in software ecosystems},
```
Comment (h3ssto) [inline literature.bib:4892]: {{...}} + capitalization

File: literature.bib [hunk 2]
```diff
@@ -6284,6 +6330,25 @@ @article{TKB+:SCP14
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2013 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@book{JCB13,
+	title = {{Software Ecosystems: Analyzing and Managing Business Networks in the Software Industry}},
+	author = {Jansen, Slinger and Cusumano, Michael A and Brinkkemper, Sjaak},
+	year = 2013,
+	publisher = {Edward Elgar Publishing}
```
Comment (h3ssto) [inline literature.bib:6337]: Address missing.

File: literature.bib [hunk 3]
```diff
@@ -6284,6 +6330,25 @@ @article{TKB+:SCP14
...
+}
+
+@article{MH:JSS13,
```
Comment (h3ssto) [inline literature.bib:6340]: publisher + address missing

File: literature.bib [hunk 4]
```diff
@@ -7820,6 +7885,43 @@ @inproceedings{RAK+:VaMoS13
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2012 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{H:JSS12,
```
Comment (h3ssto) [inline literature.bib:7888]: publisher + address missing

File: literature.bib [hunk 5]
```diff
@@ -7820,6 +7885,43 @@ @inproceedings{RAK+:VaMoS13
...
+}
+
+@article{BWB:BISE12,
+	title = {{Software Ecosystems}},
+	author = {Burkard, Christoph and Widjaja, Thomas and Buxmann, Peter},
+	journal = BISE,
+	volume = {4},
+	number = {1},
+	pages = {41--44},
+	publisher = Springer,
```
Comment (h3ssto) [inline literature.bib:7921]: Address missing, especially in case of Springer, this is an important piece of information.

File: literature.bib [hunk 6]
```diff
@@ -11483,6 +11585,19 @@ @article{BSRC10
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2009 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+% According to dl.acm.org the publisher is the Carnegie Mellon University and not ACM.
```
Comment (h3ssto) [inline literature.bib:11588]: According to IEEE, dblp and others (as well as the Proceedings front matter...) publisher is ACM

File: literature.bib [hunk 7]
```diff
@@ -14301,6 +14416,14 @@ @inproceedings{YGM:SEAS06
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2005 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@book{MS05,
+  title = {{Software Ecosystem: Understanding an Indispensable Technology and Industry}},
+  author = {{Messerschmitt, David G and Szyperski, Clemens}},
+  volume = {1},
+  year = 2005,
+  publisher = MITpress
```
Comment (h3ssto) [inline literature.bib:14424]: address missing

**Review by h3ssto — CHANGES_REQUESTED** [review]

File: literature.bib (hunk 3 above)
Comment (tthuem) [reply literature.bib:6340]: it is uncommon to specify publisher and address for journal articles

Comment (pmbittner) [reply literature.bib:6340]: Ok so I won't fix.

File: literature.bib (hunk 4 above)
Comment (pmbittner) [reply literature.bib:7888]: won't fix; see above

File: literature.bib (hunk 5 above)
Comment (pmbittner) [reply literature.bib:7921]: won't fix because journal as above; also I was not able to find the address

File: literature.bib (hunk 7 above)
Comment (pmbittner) [reply literature.bib:14424]: There are two addresses in the book direcltly next to each other, `CambridgeMIT` and `London`. What is out policy in such a case?

File: literature.bib (hunk 6 above)
Comment (pmbittner) [reply literature.bib:11588]: But the other entries for SPLC09 also have publisher `SEI` which seems actually do be the dedicated variable I should have used here in the first place. One entry even has Springer listed as publisher but I have no idea why.

Comment (h3ssto) [reply literature.bib]: There are conferences that switched their publisher multiple times.

Comment (pmbittner) [reply literature.bib]: Yes, but for the same converence (SPLC 2009)? I changed the publisher now to `SEI` here,

File: literature.bib (hunk 3 above)
Comment (h3ssto) [reply literature.bib:6341]: That is one way to handle it, one the other hand one could argue that we have many fields that are uncommon (i.e. not filled in by dblp etc.). It is my opinion that the 5 minutes of extra efforts are worth it.

Comment (pmbittner) [reply literature.bib:6341]: I will add Elsevier as publisher but I have no idea where to find the address (within 5 minutes) as even the publishers themselves don't make this information public.

File: literature.bib (hunk 7 above)
Comment (pmbittner) [reply literature.bib:14426]: I just pick `CambridgeMIT` now as both are fine.

File: literature.bib (hunk 3 above)
Comment (pmbittner) [reply literature.bib:6341]: The addresses are even filtered out when generating `literature-cleaned.bib`. So I don't see a reason here to spend more time searching for the needle in the haystack for this publication and all those to come in the next years.

**Pull request merged**